packages/web/src/app/Header.tsx

```diff
@@ -1,85 +1,47 @@
 } from '../shared/components/actions/DropdownMenu';
 import { Icon } from '../shared/components/Icon';
 import apiClient from '../shared/apiClient';
 import useAuth from '../shared/hooks/useAuth';
 import { Link } from 'react-router-dom';
 import { useAccessControl } from '../shared/accessControl';
 
-export interface HeaderProps {
-  language: string;
-  onLanguageChange(code: string): void;
-}
-
-export default function Header({ language, onLanguageChange }: HeaderProps) {
+export default function Header() {
   const session = useAuth();
   const { t } = useTranslation(['translation', 'users']);
 
-  const languagesQuery = useQuery(['languages'], () =>
-    apiClient.languages.findAll()
-  );
-  const translationLanguages = languagesQuery.data?.data ?? [];
-  const selectedLanguage = translationLanguages.find(
-    (l) => l.code === language
-  );
-
   const userCan = useAccessControl();
 
   return (
-    <header className="p-2 flex items-baseline flex-row z-10">
+    <header className="py-2 px-4 flex items-baseline flex-row z-10 mb-2">
       <Link className="font-bold text-lg" to="/">
         {t('app_name')}
       </Link>
       <div className="flex-grow" />
       <nav className="flex items-baseline" aria-label="primary">
-        {translationLanguages.length > 0 && (
-          <>
-            <Link
-              to={'/translate'}
-              className="me-4 focus:outline-none hover:underline focus:underline"
-            >
-              {t('translate')}
-            </Link>
-            {userCan('administer', 'Language') ? (
-              <DropdownMenu
-                text={selectedLanguage?.name ?? 'Language'}
-                className="me-4"
-              >
-                <DropdownMenuSubmenu text={t('switch_language')}>
-                  {translationLanguages.map((language) => (
-                    <DropdownMenuButton
-                      key={language.code}
-                      onClick={() => onLanguageChange(language.code)}
-                    >
-                      {language.name}
-                    </DropdownMenuButton>
-                  ))}
-                </DropdownMenuSubmenu>
-                <DropdownMenuLink to={'/languages'}>
-                  {t('manage_languages')}
-                </DropdownMenuLink>
-              </DropdownMenu>
-            ) : (
-              <DropdownMenu
-                text={selectedLanguage?.name ?? 'Language'}
-                className="me-4"
-              >
-                {translationLanguages.map((language) => (
-                  <DropdownMenuButton
-                    key={language.code}
-                    onClick={() => onLanguageChange(language.code)}
-                  >
-                    {language.name}
-                  </DropdownMenuButton>
-                ))}
-              </DropdownMenu>
-            )}
-          </>
+        <Link
+          to={'/translate'}
+          className="me-4 focus:outline-none hover:underline focus:underline"
```

Comment:
What is the benefit of `me-4` repeatedly applied, rather than just `gap-4` on the parent? To me it seems like a lot of duplicate code.